addons/point_of_sale/tests/test_pos_basic_config.py

```diff
@@ -0,0 +1,256 @@
+import odoo
```

Comment:
you can remove all those tests, everything is already checked in tests in file: 'test_pos_products_with_tax.py'